dataverse-persistence/src/main/resources/db/migration/V44__addTablesForROR.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TABLE IF NOT EXISTS rordata (
+    id BIGSERIAL PRIMARY KEY,
+    rorid CHAR(9) UNIQUE,
```

Comment:
If this is only 9 in length I would also reflect it in Entity (@Column(length = 9, unique = true)